interop/src/h3_server.rs

```diff
@@ -55,29 +54,39 @@ async fn main() -> Result<()> {
     let cert_chain =
         quinn::CertificateChain::from_certs(vec![quinn::Certificate::from_der(&cert_chain)?]);
 
-    let mut server = ServerBuilder::default();
-    server
-        .certificate(cert_chain, key)
-        .expect("failed to add cert");
+    let mut server_config = quinn::ServerConfigBuilder::default();
+    server_config.certificate(cert_chain, key)?;
+    server_config.protocols(&[quinn_h3::ALPN, b"hq-27"]);
+    let mut endpoint_builder = quinn::Endpoint::builder();
+    endpoint_builder.listen(server_config.build());
 
-    let (_, mut incoming) = server.build().expect("bind failed");
+    let listen = "[::]:4433".parse()?;
+    let (_, mut incoming) = endpoint_builder.bind(&listen)?;
 
     println!("server listening");
     while let Some(connecting) = incoming.next().await {
         tokio::spawn(async move {
-            if let Err(e) = handle_connection(connecting).await {
-                error!("handling connection failed: {:?}", e)
+            let protos = connecting.authentication_data().protocol.unwrap();
+            println!("server received connection");
+
+            if protos == b"h3-27" {
+                if let Err(e) = handle_connection(connecting).await {
```

Comment:
Shall we rename `handle_connection()` to `h3_handle_connection()`, for symmetry with `hq_handle_connection()`? (If I were nitpicking I'd also rename them `handle_{h3,hq}_connection()` rather than having the protocol prefix up front.)

Reply:
Got them renamed. 

I prefer to use prefix as kind of a namespace to highlight which part of the program this belongs.